src/web_app/src/types/d3Types.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+export namespace d3Types {
```

Comment:
Typescript modules are already namespaces, this is why the linter discourage the use of them. I am sure this is an easy fix, correct me if I'm wrong.

Reply:
Namespaces are internal modules but yeah i checked and the preferred syntax now is the import/export. I will change the syntax.